js/views/attachment_view.js

```diff
@@ -1,271 +1,290 @@
-/*
- * vim: ts=4:sw=4:expandtab
- */
+/* eslint-env browser */
```

Comment:
Apologies for the diff noise: I enabled ESLint for the whole file as you originally suggested but before we had the discussion to do these syntactic changes in a separate PR. I’ll point out the non-trivial changes I had to make below.